### DISCUSSION TITLE
¿Las tribus tienen que ser las mismas para todo el semestre?

### DISCUSSION THREAD

**Discussion by AlejandraMichellx** [hackatomik/SISCOx256#25] [General | Lab]
Si deseo realizar el proyecto final con otra tribu, ¿no habría ningún dilema con ello?

Dado que se pueda, de cuantas personas se podría hacer el proyecto?

**Comment by quantumquirkxyz** [comment]
Bueno, lo ideal es que tengas una sola tribu para todo el semestre y así tener un mayor control y revisión de tribus. 

Las tribus son de 5 integrantes.

**Reply by AlejandraMichellx** [reply]
Vale gracias
